platform_versioned_docs/version-23.4/secrets/overview.mdx

```diff
@@ -75,26 +75,26 @@ The ECS Agent uses the [Batch Execution role](https://docs.aws.amazon.com/batch/
 **IAM permissions**
 
 1. Add the [`AmazonECSTaskExecutionRolePolicy` managed policy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonECSTaskExecutionRolePolicy.html).
-2. Add this inline policy:
+1. Add [this inline policy](https://github.com/seqeralabs/nf-tower-aws/blob/master/launch/secrets-policy-execution-role.json):
```

Comment:
Why the link if the contents of the linked file are different to the inline policy pasted below?

Might be worth mentioning to explicitly change `<YOUR_COMPUTE_REGION>` to the specific compute region that you care about.